module/VuFind/src/VuFind/Cover/Router.php

```diff
@@ -76,12 +88,37 @@ public function getUrl(RecordDriver $driver, $size = 'small')
             return false;
         }
 
-        // Array?  It's parameters to send to the cover generator:
+        // Array? It's parameters to send to the cover generator:
```

Comment:
Should we reverse the logic here, so if `$thumb` is not an array, we return it immediately? Otherwise, I think we could potentially override a URL specified by the driver as a string with a direct URL fetched from the cover services, which would be a change in behavior. Currently, a string coming from getThumbnail always overrides dynamic lookups.

Reply:
It looks like this older comment may still need to be addressed. If you like, I can set up and test some scenarios, and make adjustments if necessary; just curious if you have any opinions before I do.

Reply:
Never mind, I see that you did address this after all. :-)